### PR TITLE
Add fromString on ByteString

### LINKED ISSUE #12: Dangerous: fromString (instance IsString ByteString)
It silently truncates Unicode characters: https://twitter.com/ChShersh/status/1442800460268720128

(I volunteer to add this to the list but it will take some time.)

### PR DESCRIPTION
**Pull request by tomjaguarpaw**
Closes https://github.com/NorfairKing/haskell-dangerous-functions/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -592,8 +603,6 @@ Prelude> putStrLn "\973"
 
 Use `ByteString`-based output, on encoded `Text` values or directly on bytestrings instead.
 
-`writeFile` caused a real-world outage for @tomjaguarpaw on 2021-09-24.
```
Comment (NorfairKing) [inline README.md]: Please don't undo this change..?

Comment (tomjaguarpaw) [reply README.md]: OK done.
